templ_attr.cif

```diff
@@ -661,11 +661,15 @@ save_
 
 save_cromer_mann_coeff
 
-    _definition.update           2012-11-29
+    _definition.update           2023-06-03
     _description.text
 ;
      The set of data items used to define Cromer-Mann coefficients
-     for generation of X-ray scattering factors.
+     for generation of X-ray scattering factors (0.0 < s < 2.0).
+
+    f(s) = c + Sum( a~i~ * exp(-b~i~ * s^2^), i=1:4)
+
+    where s = sin(theta)/lambda.
```

Comment:
Seems like a very nice clarification. Maybe we could also specify somewhere that `s` is measured in reciprocal angstrom (or that lambda is measured in angstrom)?

Reply:
I'll do lambda is measured in angstrom